asllib/doc/ASLSemanticsReference.tex

```diff
@@ -3459,19 +3369,13 @@ \section{SemanticsRule.SReturnSome \label{sec:SemanticsRule.SReturnSome}}
 
 \section{SemanticsRule.SSeq \label{sec:SemanticsRule.SSeq}}
   \subsection{Prose}
-  Evaluation of the statement $\vs$ in an environment $\env$ is
-  either a configuration $D$ (which is not a returning configuration),
-  a returning configuration (that is, \\ $\Returning((\vvs, \newg), \newenv)$),
-  or an abnormal configuration,
-  and all of the following apply:
+  All of the following apply:
   \begin{itemize}
     \item $\vs$ is a \emph{sequencing statement} \texttt{s1; s2}, that is, $\SSeq(\vsone, \vstwo)$;
     \item evaluating $\vsone$ in $\env$ is either $\Continuing(\vgone, \envone)$ in which case
     the evaluation continues,
-    or a returning configuration ($\Returning((\vvs, \newg), \newenv)$) or an abnormal configuration,
-    which short-circuit the entire evaluation;
-    \item evaluating $\vstwo$ in $\envone$ is either a non-abnormal configuration $C$
-    \ProseOrAbnormal;
+    or a returning configuration ($\Returning((\vvs, \newg), \newenv)$)\ProseOrAbnormal;
+    \item evaluating $\vstwo$ in $\envone$ yields a non-abnormal configuration $C$\ProseOrAbnormal;
```

Comment:
non-abnormal = normal?

Reply:
can also be continuing. I added explanation.